proton-j/src/main/java/org/apache/qpid/proton/engine/impl/EventImpl.java

```diff
@@ -68,12 +69,19 @@ void clear()
     }
 
     @Override
-    public Type getType()
+    public EventType getEventType()
     {
         return type;
     }
 
     @Override
+    public Type getType() {
+        if (type instanceof Type)
```

Comment:
Braces on if statement.

It would be nice if we didnt need to do an instanceof check, though I can only see making all non-core events implement getType() fixing that... :/